src/locales/ui/ui-es.yaml

```
@@ -1153,3 +1153,13 @@ entries:
   obtenez une estimation <1></1>de votre <3>empreinte carbone</3>.: <1></1>obtenga una estimación de su<3>huella de carbono</3>.
   En règle générale, les valeurs d'empreinte eau varient <2>entre 3 000 et 9 000 litres par jour</2>. Contrairement au carbone, il n'existe pas d'objectif chiffré pour l'empreinte eau.: Por regla general, los valores de la huella hídrica varían <2>entre 3.000 y 9.000 litros al día</2>. A diferencia del carbono, no existe un objetivo cuantificado para la huella hídrica.
   litres: litros
+  obtenez une estimation: solicitar presupuesto
```

j'imagine qu'ici aussi il vaudrait mieux utiliser :
```suggestion
  obtenez une estimation: solicitar estimación
```